### PR TITLE
Add change idle flags behavior and GM

### PR DESCRIPTION
**Pull request by aronwk-aaron**
Tested that this triggers appropriately when using things that have this behavior.
Weapons include (but are not limited to) the charge up attacks for the wormholer and the spinjutsu weapons

### CODE REVIEW

File: dGame/dGameMessages/GameMessages.h [hunk 1]
```diff
@@ -373,6 +374,15 @@ namespace GameMessages {
...
 	void SendDisplayChatBubble(LWOOBJID objectId, const std::u16string& text, const SystemAddress& sysAddr);
 
+	/**
+	 * @brief
+	 *
+	 * @param objectId ID of the entity to set idle flags
+	 * @param FlagsOn Flag to turn on
+	 * @param FlagsOff Flag to turn off
+	 */
+	void SendChangeIdleFlags(LWOOBJID objectId, eAnimationFlags FlagsOn, eAnimationFlags FlagsOff, const SystemAddress& sysAddr);
```
Comment (EmosewaMC) [inline dGame/dGameMessages/GameMessages.h:384]: These default to zero for what its worth per [the struct](https://lcdruniverse.org/lu_packets/src/lu_packets/world/gm/client/mod.rs.html#1363-1368)

**Review by EmosewaMC — APPROVED** [review]
How does this interact if you have multiple idle flags active at once?  Is the behavior as expected?

File: dGame/dGameMessages/GameMessages.h (hunk 1 above)
Comment (aronwk-aaron) [reply dGame/dGameMessages/GameMessages.h:384]: Ty will make it default to 0

**Comment by aronwk-aaron** [comment]
> How does this interact if you have multiple idle flags active at once?  Is the behavior as expected?

There's not any combination of skills that allow it to be applied multiple at the same time. This is exclusively used on two handed weapons as far as I can tell. I'll double check that later though

**Review by EmosewaMC — APPROVED** [review]

**Review by Wincent01 — APPROVED** [review]

**Pull request merged**